scripts/retrieveEnvVars.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
```

Comment:
Hi @xiyan128 , I rewrote the env script with the nodejs script because I'm not quite familiar with shell syntax and the new implementation is much complex than the previous version. Hence I migrated your initial script to the nodejs.